src/components/sandbox/page.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { Box, Flex, Button } from '@blockstack/ui';
+import { useConnect } from '@blockstack/connect';
+import { useHover } from 'use-events';
+
+import { truncateMiddle, microToStacks } from '@common/utils';
+import { useUserSession } from '@common/hooks/use-user-session';
+import { useDebugState } from '@common/sandbox';
+import { Tabs } from '@components/sandbox/tabs';
+import { Text, Title } from '@components/typography';
+import { PageWrapper } from '@components/page';
+import { Card } from '@components/card';
+import { Meta } from '@components/meta-head';
+
+const SignedOutView = ({ onClick }: any) => {
```

Comment:
Ensures it's typed as a fn that return `any`

```suggestion
const SignedOutView = ({ onClick }: { onClick: () => any }) => {
```

Reply:
This is updated/changed in a different branch, will avoid changing here.